doc/lightweight-styling.md

```diff
@@ -0,0 +1,60 @@
+# Lightweight Styling
+
+[Lightweight styling](https://learn.microsoft.com/windows/apps/design/style/xaml-styles#lightweight-styling) is a way to customize the appearance of XAML controls by **overriding** the system brushes that they use. Lightweight styling is applied in the Resources of the control itself:
+
+```xml
+<Button Content="Default Button Style" Style="{StaticResource MaterialFilledButtonStyle}" />
+
+<Button Content="Overridden Button Style" Style="{StaticResource MaterialFilledButtonStyle}" BorderThickness="2">
+	<Button.Resources>
+		<SolidColorBrush x:Key="FilledButtonForeground" Color="DarkGreen" />
+		<SolidColorBrush x:Key="FilledButtonBackground" Color="LightGreen" />
+		<SolidColorBrush x:Key="FilledButtonBorderBrush" Color="DarkGreen" />
+	</Button.Resources>
+</Button>
+```
+
+![Material - Button lightweight styling](assets/material-lightweight-styling-anatomy.png)
+
+Most controls have multiple states, such as **PointerOver** (mouse is hovered over), **Pressed** (control is pressed on), and **Disabled** (control is not interactable). These states are appended onto the endings of the resource keys: ButtonForeground*PointerOver*, ButtonForeground*Pressed*, and ButtonForeground*Disabled*. Some controls even have **Checked** and **Unchecked** states.
```

Comment:
Add details and a link about where to find all the available states for each control.

Reply:
thanks for the feedback! I changed the wording to "All interactive controls have multiple states", because only the ProgressBar and ProgressRing do not. As for the Checked and Unchecked states, only the CheckBox and RadioButton have these states. 

This adds slightly more to maintain documentation wise if ever we add more controls in the future that may influence the wording, but I think if the reader wants more details about which resources have PointerOver, Pressed and Disabled states for a specific control, they can follow the links at the bottom of the page.